plaso/parsers/sqlite_plugins/windows_timeline.py

```diff
@@ -0,0 +1,218 @@
+          '[OriginalPayload] BLOB, [OriginalLastModifiedOnClient] DATETIME, '
+          '[ETag] INT NOT NULL)'),
+      'Activity_PackageId': (
+          'CREATE TABLE [Activity_PackageId]([ActivityId] GUID NOT NULL, '
+          '[Platform] TEXT NOT NULL, [PackageName] TEXT NOT NULL, '
+          '[ExpirationTime] DATETIME NOT NULL)'),
+      'AppSettings': (
+          'CREATE TABLE [AppSettings]([AppId] TEXT PRIMARY KEY NOT NULL, '
+          '[SettingsPropertyBag] BLOB, [AppTitle] TEXT, [Logo4141] TEXT)'),
+      'ManualSequence': (
+          'CREATE TABLE [ManualSequence]([Key] TEXT PRIMARY KEY NOT NULL, '
+          '[Value] INT NOT NULL)'),
+      'Metadata': (
+          'CREATE TABLE [Metadata]([Key] TEXT PRIMARY KEY NOT NULL, [Value] '
+          'TEXT)')}]
+
+  def ParseGenericRow(
+      self, parser_mediator, query, row, **unused_kwargs):
+    """Parses a generic windows timeline row.
+
+      Args:
+      parser_mediator (ParserMediator): mediates interactions between parsers
+        and other components, such as storage and dfvfs.
+      query (str): query that created the row.
+      row (sqlite3.Row): row.
+    """
+    query_hash = hash(query)
+
+    event_data = WindowsTimelineGenericEventData()
+
+    payload_json_string = str(self._GetRowValue(query_hash, row, 'Payload'))
+    appid_entries_string = str(self._GetRowValue(query_hash, row, 'AppId'))
+
+    # Both Payload and AppId are serialized as JSON blobs in the DB
+    payload = json.loads(payload_json_string)
+    appid_entries = json.loads(appid_entries_string)
+
+    # Attempt to populate the package_id field by checking each of these fields
+    # in the AppId json
+    package_id_locations = ['packageId', 'x_exe_path', 'windows_win32',
+                            'windows_universal', 'alternateId']
+    for location in package_id_locations:
+      for entry in appid_entries:
+        if entry['platform'] == location and entry['application'] != '':
+          event_data.package_id = entry['application']
+          break
+        # We haven't found this field, or the entry wasn't populated. Fall
+        # back to the next option
+      if event_data.package_id != None:
+        # package_id has been populated and we're done
+        break
+
+    if 'description' in payload:
+      event_data.description = payload['description']
+    else:
+      event_data.description = ''
+
+    if 'appDisplayName' in payload and payload['appDisplayName'] != '':
+      event_data.application_display_name = payload['appDisplayName']
+    elif 'displayText' in payload and payload['displayText'] != '':
+      # Fall back to displayText if appDisplayName isn't available
+      event_data.application_display_name = payload['displayText']
+
+    timestamp = self._GetRowValue(query_hash, row, 'StartTime')
+    date_time = dfdatetime_posix_time.PosixTime(timestamp=timestamp)
+    event = time_events.DateTimeValuesEvent(
+        date_time, definitions.TIME_DESCRIPTION_CREATION)
```

Comment:
Is creation time the right timestamp definition here? Isn't this execution time, or something similar?